src/components/FilterCollapse.js

```diff
@@ -130,7 +130,7 @@ class FilterCollapse extends Component {
     // Advanced Filter change. Render to filter input.
     const tokens = [];
     for (const name of allFields) {
-      if (this.state[name]) {
+      if (this.state[name]) {        
```

Comment:
Trailing space here, sorry for being annoying about this but this should get removed